text/0000-promote-aarch64-unknown-linux-gnu-to-tier1.md

```diff
@@ -0,0 +1,188 @@
+- Feature Name: `promote-aarch64-unknown-linux-gnu-to-tier-1`
+- Start Date: 2020-07-17
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Promote the Arm aarch64-unknown-linux-gnu Rust target to Tier-1.
+
+# Motivation
+[motivation]: #motivation
+
+Arch64-unknown-linux-gnu is [currently a Tier-2 Rust target](https://forge.rust-lang.org/release/platform-support.html#tier-2), in accordance with the target tier policy articulated [here](https://rust-lang.github.io/compiler-team/minutes/design-meeting/2019-09-20-target-tier-policy/).
+
+In the last 2 quarters, very good progress has been made in understanding and filling the gaps that remain in the path to attaining Tier-1 status for this target.
+
+As a direct result, those gaps have either already been filled or are very close to being filled.
+
+As such, this RFC aims to:
+
+- Evidence what has been done.
+
+- On the basis of that evidence propose that the proceeedings to promote the aarch64-unknown-linux-gnu target to the Tier-1 category may please be kickstarted.
+
+- Culminate in the actual promotion of the aarch64-unknown-linux-gnu target to Tier-1, including any and all of the relevant processes and actions as appropriate.
+
+Please note that the narrative here doesn't always match the RFC template so some liberties may have been taken in the expression.
+
+Please also note, by way of wilful disclosure, that this RFC's author is an employee of Arm.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+1. **In essence, the target tier policy for a Tier-1 target aims to obtain the following technical and tangible assurances:**
+
+   a. The Rust compiler and compiler tests must all build and pass reliably for the target in question.
+
+   b. All necessary supporting infrastructure, including dedicated hardware, to build and run the Rust compiler and compiler tests reliably must be available openly.
+
+   c. There must exist a robust and convenient CI integration for the target in question.
+
+2. **In addition, the target tier policy for a Tier-1 target aims to obtain the following stategic assurances:**
```

Comment:
stategic -> strategic?

Does this word bring any meaning, btw? Maybe just remove it

Reply:
Hi.

Thanks for your comments. I chose the word 'strategic' as it appeared to me to best describe the relevant non-technical asks in the target tier policy. I think it is important to use a discriminator for the asks in question in order to help structure the discussion. I am more than happy to use some replacement for 'strategic'. Thanks!

Reply:
I think they are noting the spelling error in "strategic": the r is missing in the RFC text.

Reply:
Agree! :) Will address. I was meaning to provide a justification for the latter part of the coment from l4l (which was 'Does this word bring any meaning....'.